rekognition_tutorial/short_sa_rekognition_tutorial.ipynb

```diff
@@ -0,0 +1,1222 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cf95a335",
+   "metadata": {},
+   "source": [
+    "![](./tutorial_images/model_ready_to_run.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0a34d75",
+   "metadata": {},
+   "source": [
+    "Since we have our model trained we can now use it to get the predictions for unlabeled data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc3e1c9c",
+   "metadata": {},
+   "source": [
+    "Before that we should start the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c9fdaa53",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def start_model(client, project_arn, model_arn, version_name, min_inference_units):\n",
+    "\n",
+    "    try:\n",
+    "        # Start the modelstart_the_model\n",
+    "        print('Starting model: ' + model_arn)\n",
+    "        response=client.start_project_version(ProjectVersionArn=model_arn, MinInferenceUnits=min_inference_units)\n",
+    "        # Wait for the model to be in the running state\n",
+    "        project_version_running_waiter = client.get_waiter('project_version_running')\n",
+    "        project_version_running_waiter.wait(ProjectArn=project_arn, VersionNames=[version_name])\n",
+    "\n",
+    "        #Get the running status\n",
+    "        describe_response=client.describe_project_versions(ProjectArn=project_arn,\n",
+    "            VersionNames=[version_name])\n",
+    "        for model in describe_response['ProjectVersionDescriptions']:\n",
+    "            print(\"Status: \" + model['Status'])\n",
+    "            print(\"Message: \" + model['StatusMessage']) \n",
+    "    except Exception as e:\n",
+    "        print(e)\n",
+    "        \n",
+    "    print('Done...')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70110780",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "start_model(rek_client,\n",
+    "            project_arn, \n",
+    "            model_arn, \n",
+    "            version_name, \n",
+    "            min_inference_units = 1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a2f699d",
+   "metadata": {},
+   "source": [
+    "After the cell above is finished we could see that model status on Rekognition page is now \"Running\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ffe04696",
+   "metadata": {},
+   "source": [
+    "![](./tutorial_images/model_is_running.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3525a10",
+   "metadata": {},
+   "source": [
+    "## Prediction test\n",
+    "<a id='predict'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35f21b17",
+   "metadata": {},
+   "source": [
+    "Now the model is ready to run and we can get prediction for any unlabeled image we have.\n",
```

Comment:
Please add some extra where in Rekognition you can see this project, this train run and the results. Don't by shy to put some images.